### PR TITLE
Remove macOS 13.2 from ansible-test

### LINKED ISSUE #82025: Remove macOS 13.2 from ansible-test
### Summary

Remove macOS 13.2 from ansible-test after a 2-week transition period following the addition of macOS 14.0 to ansible-test. This is a remote VM removal.

### Issue Type

Feature Idea

### Component Name

`ansible-test`

### PR DESCRIPTION
**Pull request by mansoor17syed**
##### SUMMARY

Remove macOS 13.2 from ansible-test And fixes https://github.com/ansible/ansible/issues/82025

##### ISSUE TYPE
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

Please let me know if anything additional to be worked on, its my first PR wanna do a lot more.

### CODE REVIEW

**Comment by mattclay** [comment]
@mansoor17syed Thanks for your interest in contributing to Ansible. Issue https://github.com/ansible/ansible/issues/82025, and others that are part of a project such as https://github.com/ansible/ansible/projects/48, are already assigned to members of the core team (although this is not always reflected on GitHub).

To help finding a good place to start contributing, I recommend looking at the [Ansible Community Guide](https://docs.ansible.com/ansible/devel/community/), if you haven't already.

File: changelogs/fragments/ansible_test_rmv_macos_132.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - remove macOS 13.2 from remotes.
```
Comment (webknjaz) [inline changelogs/fragments/ansible_test_rmv_macos_132.yml]:
```suggestion
  - ansible-test - Removed test remotes: macos/13.2
```

**Review by webknjaz — APPROVED** [review]

File: changelogs/fragments/ansible_test_rmv_macos_132.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Removed test remotes: macos/13.2
```
Comment (felixfontein) [inline changelogs/fragments/ansible_test_rmv_macos_132.yml]: This is now a list with a single-element dictionary.
```suggestion
  - "ansible-test - Removed test remotes: macos/13.2"
```

Comment (webknjaz) [reply changelogs/fragments/ansible_test_rmv_macos_132.yml]: Ah, right,thanks!

**Pull request merged**